k9mail/src/main/java/com/fsck/k9/fragment/MessageListFragment.java

```diff
@@ -2012,7 +2012,16 @@ private View getFooterView(ViewGroup parent) {
 
     private void updateFooterView() {
         if (!mSearch.isManualSearch() && mCurrentFolder != null && mAccount != null) {
-            if (mCurrentFolder.loading) {
+            int msg=100;
```

Comment:
This is a highly arbitrary figure and appears legitimate. We have a tendency to prefer obviously invalid values rather than something that could look right.